packages/ui/src/css/vars.ts

```diff
@@ -27,7 +27,7 @@ export const vars = {
 		touchable: 'transform 0.2s cubic-bezier(0.02, 1.505, 0.745, 1.235)',
 	},
 	shadows: {
-		small: '0px 2px 8px -2px rgba(59, 59, 59, 0.08)',
+		small: '0px 6px 12px -2px rgba(59, 59, 59, 0.12)',
```

Comment:
We weren't using this shadow anywhere and we were importing a shadow from the button's stylesheet a few places. Figured it made more sense to live in the main `shadows` object.